app/screens/sso/index.tsx

```diff
@@ -70,9 +70,12 @@ const SSO = ({config, serverUrl, ssoType, theme}: SSOProps) => {
         setLoginError(errorMessage);
     };
 
-    const onMMToken = async (token: string) => {
-        Client4.setToken(token);
+    const onCSRFToken = (csrfToken: string) => {
+        const client = NetworkManager.clients[serverUrl];
```

Comment:
we should pass this and the MM token to the `ssoLogin` action in order to create and configure the client there.